lib/sendcloud/errors/response_error.rb

```diff
@@ -16,6 +17,12 @@ def initialize(payload:, body:, status:)
       def build_message
         "Sendcloud error: #{status} - #{body[:error][:message]}"
       end
+
+      def sanitized_response(raw_response)
+        raw_response.to_hash.tap do |r|
+          %i[response_headers request].each { |key| r.delete(key) }
```

Comment:
I know we chatted about this and said we could filter out the response headers, but it might actually be useful to include these in case we need to check any throttling related headers.

it would however be useful to filter out the request headers so we're not logging the auth headers:
```suggestion
          %i[request_headers request].each { |key| r.delete(key) }
```